src/components/com_tjcertificate/administrator/models/fields/certificatetemplates.php

```diff
@@ -31,11 +32,17 @@ protected function getOptions()
 	{
 		$options = array();
 
-		$user = JFactory::getUser();
-		$db = JFactory::getDbo();
+		$app  = Factory::getApplication()->input;
+		$user = Factory::getUser();
+		$db   = Factory::getDbo();
 
 		$client = $this->getAttribute('client');
 
+		if (empty($client))
+		{
+			$client = $app->get('extension', '');
+		}
```

Comment:
why do we need this?

Reply:
setting client to load templates by client